client/src/Login.js

```diff
@@ -0,0 +1,115 @@
+import React, {useState, setState} from 'react';
+import { Grid, Paper } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import cloud from './images/cloud.png';
+import ui from './images/ui.png';
+import lock from './images/lock.png';
+
+const useStyles = makeStyles({
+  mainContentStyle: {
+    color: "whitesmoke",
+    fontSize: "40px",
+    textAlign: "center",
+  },
+  SubHeadings: {
+    color: "whitesmoke",
+    fontSize: "14px",
+    textAlign: "center",
+  },
+  Slogan: {
+    color: "whitesmoke",
+    fontSize: "24px",
+    textAlign: "center",
+    backgroundColor: '#2e2e2e'
+  },
+  Button: {
+    width: "50%",
+    alignItems: 'center'
+  }
+})
+
+function Login() {
+  const classes = useStyles();
+
+  return (
+    <Grid 
+      container
+      direction="row"
+      style={{height: '100%'}}>
+      <Grid 
+        item 
+        xs={4}
+        style={{height: '100%', backgroundColor: '#2e2e2e'}}>
+        <Grid 
+        container
+        direction="column"
+        style={{minWidth: '100%', height: '100%', backgroundColor: '#2e2e2e', alignItems:"center", justify:"center"}}>
+            <Grid 
+                item
+                xs={6}
+                style={{backgroundColor: '#444449', minWidth: '90%', minHeight: '90%', margin: 'auto', alignItems:"center"}}>
```

Comment:
Same as above

Reply:
Just going to imply this for the rest of the doc to keep down on repeated comments